tests/meta/workflow/test_microsalt.py

```diff
@@ -0,0 +1,216 @@
+    mocker.patch.object(TrailblazerAPI, "add_comment")
+
+    # GIVEN a store with a QC ready microsalt case that will pass QC
+    microsalt_pass_case: Family = store.family(microsalt_case_qc_pass)
+    microsalt_pass_case.samples[1].control = "negative"
+    microsalt_pass_case.samples[1].reads = 1100000
+
+    mocker.patch.object(
+        MicrosaltAnalysisAPI,
+        "get_completed_cases",
+        return_value=[microsalt_pass_case],
+    )
+    mocker.patch.object(
+        MicrosaltAnalysisAPI, "get_project", return_value=microsalt_qc_pass_lims_project
+    )
+
+    mocker.patch.object(
+        MicrosaltAnalysisAPI, "get_latest_case_path", return_value=microsalt_qc_pass_run_dir_path
+    )
+
+    # WHEN get cases to store
+    cases_to_store: List[Family] = microsalt_api.get_cases_to_store()
+
+    # THEN it should be stored
+    assert microsalt_pass_case in cases_to_store
+
+
+def test_get_cases_to_store_fail(
+    qc_microsalt_context: CGConfig,
+    caplog,
+    mocker,
+    microsalt_qc_fail_lims_project: str,
+    microsalt_case_qc_fail: str,
+    microsalt_qc_fail_run_dir_path: Path,
+):
+    """Test get cases to store for a microsalt case that fails QC."""
+
+    caplog.set_level(logging.INFO)
+    store = qc_microsalt_context.status_db
+    microsalt_api: MicrosaltAnalysisAPI = qc_microsalt_context.meta_apis["analysis_api"]
+    mocker.patch.object(MicrosaltAnalysisAPI, "create_qc_done_file")
+    mocker.patch.object(TrailblazerAPI, "set_analysis_status")
+    mocker.patch.object(TrailblazerAPI, "add_comment")
+
+    # GIVEN a store with a QC ready microsalt case that will fail QC
+    microsalt_fail_case: Family = store.family(microsalt_case_qc_fail)
+
+    mocker.patch.object(
+        MicrosaltAnalysisAPI,
+        "get_completed_cases",
+        return_value=[microsalt_fail_case],
+    )
+    mocker.patch.object(
+        MicrosaltAnalysisAPI, "get_project", return_value=microsalt_qc_fail_lims_project
+    )
+
+    mocker.patch.object(
+        MicrosaltAnalysisAPI, "get_latest_case_path", return_value=microsalt_qc_fail_run_dir_path
+    )
+
+    # WHEN get case to store
+    cases_to_store: List[Family] = microsalt_api.get_cases_to_store()
+
+    # Then it should not be stored
+    assert microsalt_fail_case not in cases_to_store
+
+
+def test_get_latest_case_path(
+    mocker,
+    qc_microsalt_context: CGConfig,
+    microsalt_case_qc_pass: str,
+):
+    """Test get_latest_case_path return the first case path and not single sample path"""
+    microsalt_api: MicrosaltAnalysisAPI = qc_microsalt_context.meta_apis["analysis_api"]
+
+    # GIVEN a case with different case paths, both single sample and case analyses
+    mocker.patch.object(MicrosaltAnalysisAPI, "get_project", return_value="ACC12345")
+    mocker.patch.object(
+        MicrosaltAnalysisAPI,
+        "get_case_path",
+        return_value=[
+            Path("tests/fixtures/analysis/microsalt/ACC12345A2_2023"),
```

Comment:
These should also use the analysis dir fixture (maybe you should make a microsalt dir fixture)

and avoid hard-coding the `/` use `,` instead